Sources/ArcGISToolkit/Components/JobManager/JobManager.swift

```diff
@@ -13,6 +13,7 @@
 
 import ArcGIS
 import BackgroundTasks
+import Combine
 import Foundation
 import OSLog
 import SwiftUI
```

Comment:
Is this still needed? Im guessing it was only used to get at `ObservableObject`. Now that `Combine` is imported, `SwiftUI` probably doesn't need to be.

Reply:
I tried removing the SwiftUI import and tried a build which failed with the following errors. Should we be importing `UIKit` instead of `SwiftUI`?

<img width="317" alt="Screenshot 2023-09-26 at 10 26 12" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/3969c254-0041-41c0-afec-37f7bf46b6cd">